ts/sagas/premiumMessages.ts

```diff
@@ -0,0 +1,58 @@
+import { select, call, take } from "typed-redux-saga/macro";
+import { NavigationActions } from "react-navigation";
+import { createAction } from "typesafe-actions";
+import NavigationService from "../navigation/NavigationService";
+import ROUTES from "../navigation/routes";
+import { isPremiumMessagesOptInOutEnabledSelector } from "../store/reducers/backendStatus";
+
+/**
+ * This selector is a mock for a future selector which
+ * will retrieve the preference the user gave from the
+ * backend.
+ */
+export const isPremiumMessagesAcceptedSelector = (): boolean | null => false;
+
+/**
+ * A mock action that will complete this saga.
+ */
+export const setPremiumMessagesAccepted = createAction(
+  "SET_PREMIUM_MESSAGES_ENABLED"
+);
```

Comment:
```suggestion
export const setPremiumMessagesAccepted = createStandardAction(
  "SET_PREMIUM_MESSAGES_ACCEPTED"
)<boolean>();
```
Since the user will express if yes or no, what about this suggestion?
Does it make sense?

Reply:
I missed that action string name in the last refactor, you're right 🙏 Since it was just a stub I omitted its value, but it's worth handling it, it should be fixed in: https://github.com/pagopa/io-app/pull/3877/commits/29d2d735d1520a7c3c3f98903d44c3b0ac2203f5